skill-tree/src/main/java/com/RDS/skilltree/Endorsement/EndorsementController.java

```diff
@@ -46,4 +47,14 @@ public ResponseEntity<GenericResponse<EndorsementDTO>> getEndorsementById(@PathV
             return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body(new GenericResponse<EndorsementDTO>(null, message));
         }
     }
+
+    @PostMapping(value="")
+    public ResponseEntity<?> postEndorsement(@RequestBody @Valid EndorsementDRO endorsementDRO) {
```

Comment:
@bhtibrewal can you use the generic response, we will have a similar response across all the APIs